README.md

```diff
@@ -1,70 +1,63 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Crypto Market Data
 
-## Available Scripts
+> A website that provides accurate data about various crypto currencies
 
-In the project directory, you can run:
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- React
+- Jest
+- Redux
+- JavaScript
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
 
-### `npm test`
+## Live Presentation 
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+[Live presentation Link](https://www.loom.com/share/8952f4813c124b4586e6722b01568fc4)
```

Comment:
- [ ] I recommend deploying the application using Heroku, Netlify, or GitHub Pages and providing the link in the README.